tools/clang/tools/libclang/dxcrewriteunused.cpp

```diff
@@ -65,32 +65,28 @@ class RewriteUnusedASTConsumer : public SemaConsumer {
 
 class VarReferenceVisitor : public RecursiveASTVisitor<VarReferenceVisitor> {
 private:
-  SmallPtrSet<VarDecl*, 128>& m_unusedGlobals;
-  SmallPtrSet<FunctionDecl*, 128>& m_visitedFunctions;
-  SmallVector<FunctionDecl*, 32>& m_pendingFunctions;
+  SmallPtrSetImpl<VarDecl*>& m_unusedGlobals;
+  SmallPtrSetImpl<FunctionDecl*>& m_visitedFunctions;
+  SmallVectorImpl<FunctionDecl*>& m_pendingFunctions;
```

Comment:
I thought I had to modify this class. In the end, no, but I can leave the few cleanups.